packages/react-storage/src/components/StorageBrowser/Views/Controls/BreadcrumbsControl.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { withBaseElementProps } from '@aws-amplify/ui-react-core/elements';
+
+import { StorageBrowserElements } from '../../context/elements';
+
+const { Button, ListItem, Nav, OrderedList, Span } = StorageBrowserElements;
+
+const BLOCK_NAME = `breadcrumbs`;
+
+export interface BreadcrumbItem {
+  label?: string;
+}
+
+export interface BreadcrumbsControlProps {
+  items?: BreadcrumbItem[];
+}
+
+export interface BreadcrumbsControl<
+  T extends Partial<StorageBrowserElements> = StorageBrowserElements,
+> {
+  /* ???? Where/how should we define these top level props */
+  ({ items }: BreadcrumbsControlProps): React.JSX.Element;
+  Container: T['Nav'];
+  Item: T['ListItem'];
+  List: T['OrderedList'];
+  BreadcrumbButton: T['Button'];
+  Separator: T['Span'];
+}
+
+const Container = withBaseElementProps(Nav, {
+  className: `${BLOCK_NAME}__container`,
+  'aria-label': `Breadcrumbs`,
```

Comment:
Just out of curiosity, does casing impact `aria-label` values?

Reply:
I usually opt for sentence case for aria-label text (It should be human readable), but I don't believe screen readers would pronounce `Breadcrumbs` vs `breadcrumbs` differently, or `Refresh list` vs `refresh list`.

Reply:
Makes sense to me!